From 9073dbadf82c63b65c25d59bf0a167cde17bb75c Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Fri, 5 Oct 2018 00:05:06 +0300
Subject: [PATCH 13/21] legacy: Resolve hal_gnss_default denial

* This comes up when switching gnss to hwbinder
* Mimic the old location rule to resolve the denial

Change-Id: I118ac5fad75f7ab02ccf4a728c2bc67a9eb57752
---
 legacy-common/hal_gnss_default.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/legacy-common/hal_gnss_default.te b/legacy-common/hal_gnss_default.te
index bac5a5b..c10007b 100644
--- a/legacy-common/hal_gnss_default.te
+++ b/legacy-common/hal_gnss_default.te
@@ -1 +1,2 @@
 allow hal_gnss_default location_data_file:dir create;
+allow hal_gnss_default location_data_file:fifo_file create_file_perms;
-- 
2.17.1

